microsoft-edge/devtools-guide-chromium/dom/index.md

```diff
@@ -335,7 +338,7 @@ If you want to pause when a particular node is removed, use node removal breakpo
 
 1. Select **Resume Script** (![Resume Script](./index-images/resume-script-icon.png)).
 
-### Break on subtree modifications
+#### Break on subtree modifications
 
 After you put a subtree modification breakpoint on a node, DevTools pauses the page when any of the descendants of the node are added or removed.
 
```

Comment:
359: change from:
See [Elements panel keyboard shortcuts](../shortcuts/index.md#elements-tool-keyboard-shortcuts).
to:
See [Elements tool keyboard shortcuts](../shortcuts/index.md#elements-tool-keyboard-shortcuts).